### ISSUE TITLE
Epoch timer counts up for some users

### ISSUE THREAD

**Issue by 0xKang** [vertekfi/dex-frontend#53]
Counting up for some users:
![image](https://user-images.githubusercontent.com/106936065/216208351-ba22b30c-3dd4-4d67-8d71-e98d3dffe521.png)

Counting down for some users:
![image](https://user-images.githubusercontent.com/106936065/216208563-f6ca1842-cb62-4c5b-b15e-a4249641c720.png)

The Voting Period Ends box with the timer counts upwards for some users, but down from 7 days for others. I think this issue was encountered on aequinox before too. Probably the same fix. Adaptive timezone thing?

**Comment by harleen-m** [comment]
I'm too tired to say this with 100% certainty, but I believe the issue is here:
https://github.com/vertekfi/dex-frontend/blob/351bc80fee0de4c7e21d69a49d8f21f1bbb0cbad/lib/global/gauges/useVotingGauges.tsx#L96-L100

If the user is in UTC+ everything works as expected.
If the user is in UTC- the countdown will be a "countup" until the next day (Thursday) in their timezone.

Let's say I'm in UTC on Wednesday; next Thursday will be tomorrow.
Let's say I'm in UTC on Thursday; next Thursday will be next week.
Let's say I'm in EST on Wednesday; next Thursday will be tomorrow.
**Let's say I'm in EST on Wednesday 8pm (Thursday 1am UTC); next Thursday is still tomorrow, but in UTC it's already today.**
Let's say I'm in EST on Thursday; next Thursday will be next week.

Long story short we cannot combine UTC and local time and expect the math will just work out. I don't know the `date-fns` library, but isn't there a function for "Next UTC Thursday"? Or am I overthinking it?

**Comment by 0xKang** [comment]
This seems to have been resolved in a recent update. Will monitor for any funky timezone stuff. But thank you Harleen for the above.

**Comment by harleen-m** [comment]
I don't see how it has been resolved. I think it'll happen again next Wednesday/Thursday. Let's see. ⏳

**Comment by harleen-m** [comment]
![image](https://user-images.githubusercontent.com/119888996/217787009-85a22010-a244-4e0e-81d9-b40f9fbb0b70.png)
